scrapy/extensions/feedexport.py

```diff
@@ -235,5 +224,33 @@ def _get_uri_params(self, spider):
             params[k] = getattr(spider, k)
         ts = datetime.utcnow().replace(microsecond=0).isoformat().replace(':', '-')
         params['time'] = ts
+        params['start'] = self.start
         self._uripar(params, spider)
         return params
+
+    def _start_export(self, spider):
```

Comment:
exporters have a `start_exporting` method, maybe call it `_start_exporting` as well? Or `_start_new_batch`? Please ignore me if you think `_start_export`  is fine :)